cura/Settings/MachineManager.py

```diff
@@ -405,12 +424,12 @@ def clearUserSettingAllCurrentStacks(self, key):
         for container in send_emits_containers:
             container.sendPostponedEmits()
 
-    ##  Check if the global profile does not contain error states
-    #   Note that the _active_stack_valid is cached due to performance issues
-    #   Calling _checkStackForErrors on every change is simply too expensive
-    @pyqtProperty(bool, notify = activeStackValidationChanged)
-    def isActiveStackValid(self):
-        return bool(self._active_stack_valid)
+    ##  Check if none of the stacks contain error states
+    #   Note that the _stacks_valid is cached due to performance issues
+    #   Calling _checkStack(s)ForErrors on every change is simply too expensive
+    @pyqtProperty(bool, notify = stacksValidationChanged)
+    def areStacksValid(self):
```

Comment:
While this just changes the original property, I think the naming of this (and also the original) is wrong. A stack with an error value is still a valid stack, it just has an error. An invalid stack would be something like a stack with a container that does not exist or other major programmatic error, not a stack with just an error value.